src/services/services.ts

```diff
@@ -5490,9 +5490,10 @@ namespace ts {
                 };
             }
 
-            function isImportOrExportSpecifierImportSymbol(symbol: Symbol) {
+            function isES6StyleImportSymbol(symbol: Symbol) {
                 return (symbol.flags & SymbolFlags.Alias) && forEach(symbol.declarations, declaration => {

```

Comment:
Since you only use `kind`, can you just declare a variable named `kind` or destructure it from the parameter?